### ISSUE TITLE
fix(List): ListItemGraphic and ListItemMeta missing aria-hidden

### ISSUE THREAD

**Issue by mgr34** [jamesmfriedman/rmwc#575]
ListItemGraphic and ListItemMeta when a graphic should include `aria-hidden=true`
